samples/benchmarks/coremark/Kconfig.sysbuild

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+source "share/sysbuild/Kconfig"
+
+config IMAGE_2_BOARD
+	string "Second additional board name to build sample for"
+	default "nrf5340dk_nrf5340_cpunet" if $(BOARD) = "nrf5340dk_nrf5340_cpuapp"
+	default "nrf5340dk_nrf5340_cpunet" if $(BOARD) = "nrf5340dk_nrf5340_cpuapp_ns"
+
+config IMAGE_3_BOARD
```

Comment:
Consider: I guess we don't need this currently? Maybe we could introduce it back when the third image will be needed?